src/picongpu/include/plugins/radiation/Radiation.hpp

```diff
@@ -726,9 +726,9 @@ class Radiation : public ISimulationPlugin
       this->currentStep = currentStep;
 
       /* the parallelization is ONLY over directions:
-       * (a combinded parallelization over direction AND frequencies
-       *  turned out to be slower on fermis (couple percent) and
-       *  definitly slower on kepler k20)
+       * (a combined parallelization over direction AND frequencies
+       *  turned out to be slower on Fermis (couple percent) and
```

Comment:
Fermis is internal slang, maybe better use "GPUs of the Fermi generation (sm_2x)"